source/_docs/guides/bitbucket.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Continuous Integration with Bitbucket and CircleCI
+description: How to integrate Pantheon with CircleCI and Bitbucket.
+tags: [siteintegrations]
+type: guide
+permalink: docs/:basename/
+draft: true
+date: 4/25/2017
+h
+---
+
+## CircleCI
+
+Continuous Integration services such as [CircleCI](https://circleci.com) allow developers to automate the process of testing code. This insures that, if a thorough testing suite is defined, errors are caught before the code is deployed.
+
+## What you will build
+
+In this guide, we will use [Composer](https://getcomposer.org) to build a project containing the configuration for Drupal 8, [Drush](http://docs.drush.org/en/master), a simple example of a Behat [test suite](https://github.com/pantheon-systems/example-drops-8-composer/tree/master/tests) and other required configuration files. We will move this to a Bitbucket repository and configure CircleCI.
+
+When code is submitted for deployment, CircleCI will create a testing environment and run a suite of tests. If tests pass, the code can be merged into the master branch of the codebase. Before this merge happens, CircleCI will repeat the testing process against the production branch.
+ 
+
+## What you’ll need
+
+- Composer, git and [Terminus](https://pantheon.io/docs/terminus) running on your local computer.
+- A Bitbucket account
+- A [Multidev](https://pantheon.io/docs/multidev/) enabled site on Pantheon. [Pantheon for Agency](https://pantheon.io/agencies/pantheon-for-agencies) signup (free) will enable Multidev.
+- A free [CircleCI](https://circleci.com) account
+
+
+## Step 1: Create the Example Project and Add Version Control
+
+1. From your terminal, use composer to make a new local project, called "bitbucket," based on our example repo:
+
+```nohighlight
+composer create-project pantheon-systems/example-drops-8-composer bitbucket
+```
+
+2. Important Note: Currently, a plugin used to manage multidev environments is only configured to work with GitHub. While a fix will be released shortly in a future version, tests will fail because of this build step. At this point, now would be a good time to comment out the following line in the ```bitbucket/circle.yml``` file we just cloned:
```

Comment:
Thanks for the feedback @greg-1-anderson! What is the timeline on #1?
Regarding #2, Are you in any position to work together on this? Your help would probably be the difference in doing this in a day or a week or more if I do it myself (I'm guessing). I can work on your time one day this week.